packages/reaction-core/common/methods/cart.js

```diff
@@ -4,8 +4,9 @@
 //
 if (Meteor.isClient) {
   Meteor.methods({
-    "cart/submitPayment": function (paymentMethod) {
+    "cart/submitPayment": function (paymentMethod, sessionId) {
```

Comment:
Because this  defined in a `isClient` block , `sessionId` could just be a `Session.get('sessionId');`